SingularityService/src/main/java/com/hubspot/singularity/data/SingularityValidator.java

```diff
@@ -230,11 +230,17 @@ public SingularityRequest checkSingularityRequest(
       !request.getInstances().isPresent() || request.getInstances().get() > 0,
       "Instances must be greater than 0"
     );
+
+    // check if requested number of instances exceeds max scale
+    int maxInstances = request.getMaxScale().isPresent()
```

Comment:
This should be the lower of the two config values (i.e. a user shouldn't be able to exceed our global max scale by specifying a request level max scale)

Reply:
To confirm, does this mean global does not automatically override request-level and vice versa, and we instead want to always use the lower of the two when validating scale?

Reply:
Yes, we should always use the lowest. Users should be able to further restrict the global option with a request level option, but not the other way around.